turbo/engineapi/engine_server.go

```diff
@@ -468,14 +474,6 @@ func (s *EngineServer) forkchoiceUpdated(ctx context.Context, forkchoiceState *e
 
 	if payloadAttributes != nil {
 		timestamp := uint64(payloadAttributes.Timestamp)
-		// op-node sends V3 before Cancun, without beacon root.
-		// so we must allow V3 before cancun to support forkChoiceUpdated backward compatibility
-		//if !s.config.IsCancun(timestamp) && version >= clparams.DenebVersion { // V3 before cancun
-		//	if payloadAttributes.ParentBeaconBlockRoot == nil {
-		//		return nil, &rpc.InvalidParamsError{Message: "Beacon Root missing"}
-		//	}
-		//	return nil, &rpc.UnsupportedForkError{Message: "Unsupported fork"}
-		//}
```

Comment:
Why did you remove this? this issue was fixed in the [op-node](https://github.com/ethereum-optimism/optimism/releases/tag/v1.5.1) and we should restore this check.